puresec_cli/actions/generate_roles/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .generate_roles import GenerateRoles as Action
+
```

Comment:
Looks like it's fine doing this -
https://docs.python.org/3/tutorial/modules.html#intra-package-references